suse_migration_services/prechecks/pre_checks.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2021 SUSE Linux LLC.  All rights reserved.
+#
+# This file is part of suse-migration-services.
+#
+# suse-migration-services is free software: you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# suse-migration-services is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with suse-migration-services. If not, see <http://www.gnu.org/licenses/>
+#
+import os
+import configparser
+
+# project
+from suse_migration_services.fstab import Fstab
+from suse_migration_services.command import Command
+
+
+def check_remote_repos():
+    repos_path = '/etc/zypp/repos.d'
+    no_remote_repos = []
+    print("PEPE")
+    if os.path.exists(repos_path):
+        repos_list = os.listdir(repos_path)
+        config = configparser.RawConfigParser()
+        if not repos_list:
+            print('No repositories in {}'.format(repos_path))
```

Comment:
Any reason why we should not use the python logging facility ?

Reply:
Given the checks should run before booting, more likely after installation, I thought it would be ok to print it in the console